test/run

```diff
@@ -12,6 +12,18 @@ declare -a WEB_APPS=({standalone,setup,setup-requirements,django,numpy,app-home,
 test_dir="$(readlink -zf $(dirname "${BASH_SOURCE[0]}"))"
 image_dir=$(readlink -zf ${test_dir}/..)
 
+test_short_summary=''
+TESTSUITE_RESULT=0
+
+TEST_LIST="\
+test_s2i_usage
+test_docker_run_usage
+test_application
+test_application_with_user
+test_scl_variables_in_dockerfile
+test_from_dockerfile
```

Comment:
`test_from_dockerfile` uses an external repository as an application source so it does not make sense to run it more than once for each image.